library/markdown.php

```diff
@@ -15,7 +15,9 @@ function Markdown($text) {
 	$stamp1 = microtime(true);
 
 	# Read file and pass content through the Markdown parser
```

Comment:
Are you saying that I should getting rid of the file and use the parser directly in `diaspora2bb.php`?